benchmarks/benchmarking.md

```diff
@@ -0,0 +1,103 @@
+# Benchmarking Setup Guide
+
+This document summarises the whole procedure to setup a test server for benchmarking we used to collect reports. Lookup individual tool's official documentation for more granular information.
+
+## Local Flask Server
+
+Requires python installation.
+
+1. Get flask and waitress from pip.
+
+   ```bash
+   pip install flask waitress
+   ```
+
+2. Create a new `app.py` file that contains -
+
+   ```python
+    from waitress import serve
```

Comment:
Shouldn't we ideally put the code required for local servers in the repository?

cc @dcharkes.

Reply:
Umm.. I can put that if that's what best practice is. But, any server will do the job. I just chosen this 2 servers as they are easy to setup, well known and used in production.

Should I include all the codes in the repo? (And, the resources they served too?)

Reply:
Added :)